### PR TITLE
Supporintg muilti-assignee data model

### PR DESCRIPTION
**Pull request by sanbornsen**
- This commit is to support multi-assignee data model update of core.
- This commit does not have UI implementation of multi assignee.
- Updated the corresponding tests (both functional and unit) to support the updated model.

This PR is to support multi-assignee data model update in core PR - https://github.com/almighty/almighty-core/pull/571

### CODE REVIEW

**Comment by aslakknutsen** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

File: src/app/models/work-item.ts [hunk 1]
```diff
@@ -19,11 +19,11 @@ export class WorkItemAttributes {
 }
 
 export class WorkItemRelations {
-  assignee: {
+  assignees: {
     data: {
       id: string,
       type: string
-    }
+    }[]
```
Comment (joshuawilson) [inline src/app/models/work-item.ts]: Nicely done. Why doesn't comments need the array brackets?

Comment (sanbornsen) [inline src/app/models/work-item.ts]: Ohh, yes that's a mistake. Comments should also be an array.

Comment (sanbornsen) [inline src/app/models/work-item.ts]: Actually, no. Comments will have a data property which will be an array. I'll add it when I'll be working on comment resolver.

Comment (joshuawilson) [inline src/app/models/work-item.ts]: I wasn't sure if it needed it, just asking.

**Pull request merged**